spec/compiler/semantic/splat_spec.cr

```diff
@@ -817,4 +817,14 @@ describe "Semantic: splat" do
       i.should eq(4)
     end
   end
+
+  it "doesn't shift a call's location", focus: true do
+    result = semantic %(class Foo\ndef bar(x)\nbar(*{"test"})\nend\nend\nFoo.new.bar("test"))
```

Comment:
This string as a multiline heredoc would make it easier to relate the line and column numbers.